Oct 08, 2019 1:50:50 PM org.jenkinsci.remoting.engine.JnlpAgentEndpointResolver resolve
INFO: Remoting server accepts the following protocols: [JNLP4-connect, JNLP-connect, Ping, JNLP2-connect]
Oct 08, 2019 1:51:21 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:2)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:51:52 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:3)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:52:23 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:4)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:52:54 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:5)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:53:25 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:6)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:53:56 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:7)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:54:27 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:8)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:54:58 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:9)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:55:29 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:10)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:56:00 PM hudson.remoting.jnlp.GuiListener$1 run
INFO: Connecting to 10.75.19.24:46313 (retrying:11)
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

Oct 08, 2019 1:56:21 PM hudson.remoting.jnlp.GuiListener$2 run
SEVERE: Failed to connect to 10.75.19.24:46313
java.io.IOException: Failed to connect to 10.75.19.24:46313
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:242)
	at hudson.remoting.Engine.connect(Engine.java:686)
	at hudson.remoting.Engine.innerRun(Engine.java:547)
	at hudson.remoting.Engine.run(Engine.java:469)
Caused by: java.net.ConnectException: Connection timed out: connect
	at sun.nio.ch.Net.connect0(Native Method)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.Net.connect(Unknown Source)
	at sun.nio.ch.SocketChannelImpl.connect(Unknown Source)
	at java.nio.channels.SocketChannel.open(Unknown Source)
	at org.jenkinsci.remoting.engine.JnlpAgentEndpoint.open(JnlpAgentEndpoint.java:203)
	... 3 more

